src/routes/canvas/toolbar/tools/MyColorPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {
+  ColorPicker,
+  getColorFromString,
+  IColor,
+  IColorPickerStyles,
+} from '@fluentui/react';
+
+/**
+ * Styles for the ColorPicker component from Fluent UI.
+ */
+const colorPickerStyles: Partial<IColorPickerStyles> = {
+  panel: { padding: '10px 15px 0px 15px'},
+  root: {
+    minWidth: 200,
+    width: 230,
+    backgroundColor: 'rgba(255, 255, 255, 0.7)'
+  },
+  colorRectangle: { minHeight: 160, minWidth: 200},
+  table: {display: 'none'},
+  colorSquare: {height: 20, width: 25, marginLeft: 10}
+};
+
+/**
+ * Props for the MyColorPicker component.
+ */
+export interface MyColorPickerProps {
+  color: IColor;
+}
+
+/**
+ * A custom color picker component.
+ * This component uses Fluent UI's ColorPicker to allow the user to pick a color.
+ * The picked color is then passed up to the parent component through the setColor prop.
+ */
+class MyColorPicker extends React.Component<{defaultColor: string, setColor: (color: string) => void}, MyColorPickerProps> {
```

Comment:
Might want to move the `defaultColor` and `setColor` props inside the `MyColorPickerPros` interface